core/ulog/README.md

```diff
@@ -0,0 +1,49 @@
+# Ulog package
+
+ulog provides an interface and abstraction layer over the logger implementation used underneath,
+and provides simple APIs for logging. The logger is instantiated as default logger and can be configured
+Configure() API and provided yaml configuration.
```

Comment:
also missing a word. and can be configured **via the** `Configure()` API or something